app/main.py

```diff
@@ -1,8 +1,26 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        self.people.update({name: self})
 
```

Comment:
people is a class variable, you can't use it with 'self', use Person.people.update({name: self}) instead